### DISCUSSION TITLE
What's the Timeline for resolving the Ojs Standalone Issue in quarto v1.3 or higher ?

### DISCUSSION THREAD

**Discussion by KaiWaldrant** [quarto-dev/quarto-cli#7487] [Q&A] [answered]
### Description

We've encountered an issue when utilizing Ojs within our .qmd documents to create standalone HTML pages, where the input files are not included unless they are hosted on a web server. We've identified the version responsible for this issue in `quarto release v1.3.30`, and the comparison tool highlights that the Ojs runtime update is the cause  ([compare](https://github.com/quarto-dev/quarto-cli/compare/v1.3.26...v1.3.30)). Due to this problem, we've had to revert to using quarto `v1.2.280`, which means we cannot leverage the latest features for quarto and Ojs.

We have a few questions related to this issue:

1. The most pressing question: Is there an estimated timeline for resolving this issue?
2. Why is this limitation not in the documentation pages, especially when using Ojs in standalone mode with versions `v1.3.30 or higher`?
3. We at JnJ are open to discussing and potentially scheduling a meeting to address this issue, as it would greatly benefit our workflow if a solution is found.

**Comment by mcanouil** [comment]
Are you talking about the following issue?
- https://github.com/quarto-dev/quarto-cli/issues/1412

Note that, for a bug to be resolved it has to be discovered/reported, thus if the issue above is not related to your issue, then I suggest you report it using the issue template if possible.

**Comment by novotny1akub** [comment]
As suggested by @mcanouil, I reported the issue [here](https://github.com/quarto-dev/quarto-cli/issues/7671).

**Comment by mcanouil** [comment, marked as answer]
I am closing this thread in favour of the issue tracking this, subscribe to it to get updates when there will be some:

- #6371

**Comment by cscheid** [comment]
After a long discussion where we considered all of our options, we've now concluded that unfortunately this isn't a problem that we have the resources to solve. It would require a complete fork and maintenance of a new version of the observablehq source code, which is unfortunately not practical.
